packages/flutter/lib/src/gestures/events.dart

```diff
@@ -487,7 +681,47 @@ class PointerHoverEvent extends PointerEvent {
          orientation: orientation,
          tilt: tilt,
          synthesized: synthesized,
+         transform: transform,
+         original: original,
        );
+
+  @override
+  PointerHoverEvent transformed(Matrix4 transform) {
+    if (transform == this.transform) {
+      return this;
+    }
+    final Offset transformedPosition = PointerEvent.transformPosition(transform, position);
+    return PointerHoverEvent(
```

Comment:
I do not want to transform the original PointerEvent because a PointerEvent may be delivered to multiple listeners and all listeners needs the event in their local coordinate space.

Not sure how the copyWith constructor would help? I would just call copyWith from here and then the copyWith constructors would contain this ugliness of copying over the values?

Reply:
Yeah, OK.